_posts/2018-01-22-LGBTSteminar2018.md

```diff
@@ -0,0 +1,23 @@
+---
+layout:     post
+title:      LGBT STEMinar
+date:       2018-01-22 09:00
+summary:    Our lessons from the 2018 LGBT STEMinar
+category:   Conferences
+author:     alice_kaerast
+---
+
+Sky Betting & Gaming send people to a large number of different conferences each year because this is a great way to learn from our peers in other organisations.  It is incredibly valuable to see how other companies approach similar issues, especially in areas of cutting edge technology where more traditional training courses often lag behind.
+
+Whilst many of these conferences are directly-related to our day-jobs, we also attend conferences that are only tangentially related.  We have an awful lot to learn from tangential fields to our own, and so one event I recently attended was the annual LGBT Steminar hosted at the University of York’s STEM Learning Centre.  I spent the day learning about some of the latest scientific research that the LGBT+ people presenting have been involved in, as well as learning a lot about the state of LGBT representation in academia.
+
+We cannot entirely separate our work from our lives, and so it is important that we can be ourselves at work.  So many of the people at this conference lacked LGBT role models when they were growing up, and are finding themselves becoming these role models whether they like it or not.  Whether we like it or not, it is our responsibility to make this path easier for future generations.  This is a big responsibility, and we need to work together in order to do this well.  One of the common themes throughout all of the talks was the importance of being visibly out.  Whilst we cannot and should not ask individuals to come out, the act of being explicitly queer is of immense value to other LGBT+ people who are still discovering themselves.  A small number of people coming out can start to create a critical mass, making others feel safe(r) to be themselves in the workplace.
```

Comment:
> whether they like it or not.  Whether we like it or not

Reads a little awkwardly to me, though I agree with your point.

Reply:
had a go at clearing this up.